Berlin/BerOrFol117.xml

```diff
@@ -80,11 +85,262 @@ schematypens="http://relaxng.org/ns/structure/1.0"?>
+
+<msItem xml:id="p2.14">
+<locus from="294" to="303"/>
+<title type="complete" xml:lang="en" ref="LIT2366Storyo">A story of Mary and the Magi</title>
+<textLang xml:lang="gez"></textLang>
+<note>Indication for reading: <q xml:lang="gez">በጌና፡</q></note>
+</msItem>
+
+<msItem xml:id="p2.15">
+<locus from="303" to="317"/>
+<title type="complete" xml:lang="en" ref="LIT1184Baenta">Homily on Nativity by bishop Gregory [of Nazianzos]</title>
+<textLang xml:lang="gez"></textLang>
+<note>Indication for reading: <q xml:lang="gez">አመ፡ ፳፱፡ ተኅሣሥ፡ </q></note>
+</msItem>
+
+<msItem xml:id="p2.16">
+<locus from="317" to="326"/>
+<title type="complete" xml:lang="en" ref="LIT6712HomJohnScet">Homily by ʾabbā Yoḥannǝs from the monastery of Macarius</title>
+<textLang xml:lang="gez"></textLang>
+<note>Indication for reading: <q xml:lang="gez">አመ፡ ፴፡ ለተኅሣሥ፡</q></note>
+</msItem>
+
+<msItem xml:id="p2.17">
+<locus from="326" to="354"/>
+<title type="complete" xml:lang="en" ref="LIT4854Sleepers">History of the seven sleepers who were martyrs in a grotto</title>
+<textLang xml:lang="gez"></textLang>
+<note>Indication for reading: <q xml:lang="gez">አመ፲፫፡ ለጥር፡</q></note>
+</msItem>
+
+<msItem xml:id="p2.18">
+<locus from="354" to="364"/>
+<title type="complete" xml:lang="en" ref="LIT6495Zenawa">On the Annunciation</title>
+<textLang xml:lang="gez"></textLang>
+<note>Indication for reading: <q xml:lang="gez">አመ፳፱፡ ለመጋቢት፡</q></note>
+</msItem>
+
+<msItem xml:id="p2.19">
+<locus from="365" to="381"/>
+<title type="complete" xml:lang="en" ref="LIT6721ChrAnnun">Homily by John Chrysostom on the Annunciation [1]</title>
+<textLang xml:lang="gez"></textLang>
+<note>Indication for reading: <q xml:lang="gez">በሰንበተ፡ ክርስቲያን፡ ዘተኅሣሥ፡</q></note>
+</msItem>
+
+<msItem xml:id="p2.20">
+<locus from="381" to="396"/>
+<title type="complete" xml:lang="en" ref="LIT6722ChrAnnunciation">Homily by John Chrysostom on the Annunciation [2]</title>
+<textLang xml:lang="gez"></textLang>
+<note>Indication for reading: <q xml:lang="gez">በ፪፡ ሰንበት፡</q></note>
+</msItem>
+
+<msItem xml:id="p2.21">
+<locus from="396" to="413"/>
+<title type="complete" xml:lang="en" ref="LIT2107OntheA">Homily by John Chrysostom on the Annunciation [3]</title>
+<textLang xml:lang="gez"></textLang>
+<note>Indication for reading: <q xml:lang="gez">በ፫፡ ሰንበት፡ ዘታኅሣሥ፡</q></note>
+</msItem>
+
+<msItem xml:id="p2.22">
+<locus from="413" to="426"/>
+<title type="complete" xml:lang="en" ref="LIT6713DerYaq">Homily by Jacob of Sarug on the Annunciation</title>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="p2.23">
+<locus from="427" to="482"/>
+<title type="complete" xml:lang="en" ref="LIT2487Vision">Homily by Theophilus of Alexandria on the flight of Holy Family to Egypt</title>
+<textLang xml:lang="gez"></textLang>
+<note>Indication for reading: <q xml:lang="gez"></q></note>
+</msItem>
+
+  </msContents>
+
+<physDesc>
+<objectDesc form="Codex">
+<supportDesc>
+<support>
+<material key="parchment"/>
+</support>
```

Comment:
Could you give the number of leafs and their extent in the composite manuscript, and also information on foliation (if available)? In composite manuscripts, I find this always really helpful to get an impression of the structure.